http_service/bugbug_http/app.py

```diff
@@ -401,7 +449,7 @@ def batch_prediction(model_name):
     auth = headers.get(API_TOKEN)
 
     if not auth:
-        return jsonify(UnauthorizedError().dump({}).data), 401
+        return jsonify(UnauthorizedError().dump({})), 401
```

Comment:
Why this change?

Reply:
It is broken at the moment and raises `AttributeError: dict objects have no attribute 'data'`. This is because the `.data` is being called on the return value from `dump` (which is just a dict) as opposed to the return value from `jsonify` (which is a flask `Response` object).

In my testing, returning a `Response` object seems to work just as well and I thought it might be more appropriate than returning a dict here. Though maybe that's a bad assumption.